plugins/Live/tests/Integration/ModelTest.php

```diff
@@ -13,7 +13,7 @@
 use Piwik\Plugins\Live\Model;
 use Piwik\Tests\Framework\Fixture;
 use Piwik\Tests\Framework\Mock\FakeAccess;
-use Piwik\Tests\Framework\TestCase\SystemTestCase;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
```

Comment:
This test was in `System` as a `SystemTestCase` but it was actually an `IntegrationTestCase`